app/stac_api/admin.py

```diff
@@ -237,3 +237,11 @@ def get_form(self, request, obj=None, **kwargs):  # pylint: disable=arguments-di
     def href(self, instance):
         path = instance.file.name
         return build_asset_href(self.request, path)
+
+    # We don't want to move the assets on S3
+    # That's why some fields like the name of the asset are set readonly here
+    # for update operations
+    def change_view(self, request, object_id, form_url='', extra_context=None):
+        self.readonly_fields = self.get_readonly_fields(request)
+        self.readonly_fields.extend(['name', 'item', 'file'])
```

Comment:
The `file` should not be read only, we should be able to change the file content from the admin page. This does actually a new upload and is needed if we want to update an asset file from admin page.